core/drivers/fabric-driver/server/server.ts

```diff
@@ -61,16 +60,26 @@ function mockCommunication(query: query_pb.Query) {
     }, 3000);
 }
 
-// Handles communication with fabric network and sends resulting data to the relay
-const fabricCommunication = async (query: query_pb.Query, networkName: string) => {
-    // Query object from requestor
-    console.log('query', query.getRequestId());
+// Package view and send to relay
+export function packageView(
```

Comment:
If we need to export these functions, then let put these functions in `utils.ts`? Same for below export functions.

Reply:
Moved the client functions to `utils.ts` and the view packaging function to `fabric-code.ts`.